### PR TITLE
feat: add rodent breed on animal 'en' locale

### PR DESCRIPTION
**Pull request by LordFitoi**
- Added `rodent` breed on `en` locale.
- Added a list of over 150 rodent breeds.
- Added tests for `rodent` method in `animal.spec.ts.snap`

### CODE REVIEW

**Review by Shinigami92 — DISMISSED** [review]

File: src/locales/en/animal/rodent.ts [hunk 1]
```diff
@@ -0,0 +1,187 @@
+export default [
+  'Genus Bathyergus',
```
Comment (pkuczynski) [inline src/locales/en/animal/rodent.ts:2]: Sorting alphabetically would be nice!

**Review by pkuczynski — APPROVED** [review]

File: src/locales/en/animal/rodent.ts [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+  'Fukomys foxi',
+  'Fukomys ilariae',
+  'Fukomys kafuensis',
+  'Fukomys mechowii',
+  'Fukomys micklemi',
+  'Fukomys occlusus',
+  'Fukomys ochraceocinereus',
+  'Fukomys whytei',
+  'Fukomys zechi',
+  'Genus Georychus',
```
Comment (pkuczynski) [inline src/locales/en/animal/rodent.ts:28]: Why sometimes the second part of the name is lower case and sometimes upper?

**Comment by ST-DDT** [comment]
**Preview** ✅ 

grafik

**Review by ST-DDT — DISMISSED** [review]
Please sort the rodent file alphabetically.

File: src/locales/en/animal/rodent.ts (hunk 2 above)
Comment (LordFitoi) [reply src/locales/en/animal/rodent.ts:28]: I forgot to remove `Genus` from these species. "The word `Genus` is a taxonomic-rank used in the biological classification of living" (Extracted from wikipedia), the real name of these species is the second part of their names.

Let me fix it

**Comment by LordFitoi** [comment]
Sorted alphabetically 👍

**Review by ST-DDT — APPROVED** [review]

**Review by import-brain — DISMISSED** [review]

**Comment by ST-DDT** [comment]
Could you please update your branch with the latest changes from the main branch?

**Comment by LordFitoi** [comment]
Sure

**Review by ST-DDT — APPROVED** [review]

**Review by Minozzzi — APPROVED** [review]

**Review by Shinigami92 — APPROVED** [review]

**Pull request merged**